### ISSUE TITLE
How to including custom field in DjangoPatchMutation? 

### ISSUE THREAD

**Issue by geekashu** [tOgg1/graphene-django-cud#29]
I have a use-case where I need your inputs on how to handle it. Any help is highly appreciated.

Lets say I have these models -
```
class Dashboard(models.Model):
    name = models.CharField(max_length=255) 

class Widget(models.Model):
    name = models.CharField(max_length=255)
    layout = models.CharField(max_length=255)
    widget = models.ForeignKey(Dashboard) 
```

```
class DashboardUpdateMutation(DjangoPatchMutation):
    class Meta:
        model = Dashboard
        login_required = True

class DashboardMutation(graphene.ObjectType):
    update_dashboard = DashboardUpdateMutation.Field()
```

I need to update the Dashboard fields and its widgets through some custom logic. I want to write the patch mutation as follows.
```
`mutation{
  updateDashboard(id: "RGFzaGJvYXJkTm9kZTox", input: {
    name: "Main Dashboard",
    mycustomfield: "layout json",
  }) {
    dashboard {                
        id,
        name,
   }
}
}`
```
How should I add `mycustomfield` above, so that I can run the logic in its resolver? A code example would help a lot.

**Comment by tOgg1** [comment]
Hi Geekashu. Sorry about the slow response.

Adding custom fields is on the TODO-list. I'll fast track it and get it out in 0.7.0.

**Comment by tOgg1** [comment]
Closed by #42.
